src/helpers/Move.js

```diff
@@ -1 +1,339 @@
+                let randomKnight = eightPositions.filter(field => field.piece === 'N');
+                randomKnight[0].piece = '';
+
+                let newColumnFields = fields.filter(field => field.column === moveColumn);
+                let newPosition = newColumnFields.filter(field => field.row === Number(moveRow));
+                newPosition[0].piece = 'N';
+            } else if(piece === 'B'){
+               
+                let fourBlocks = fields.filter(field => ((field.columnIndex !== columnInd && field.row !== Number(moveRow))
+                                                            && (field.columnIndex < columnInd && field.row > Number(moveRow) && (field.columnIndex - columnInd) === (Number(moveRow) - field.row)))
+                                                        || ((field.columnIndex !== columnInd && field.row !== Number(moveRow))
+                                                            && (field.columnIndex > columnInd && field.row > Number(moveRow) && (field.columnIndex - columnInd) === (field.row - Number(moveRow))))
+                                                        || ((field.columnIndex !== columnInd && field.row !== Number(moveRow))
+                                                            && (field.columnIndex < columnInd && field.row < Number(moveRow) && (columnInd - field.columnIndex) === (Number(moveRow) - field.row)))
+                                                        || ((field.columnIndex !== columnInd && field.row !== Number(moveRow))
+                                                            && (field.columnIndex > columnInd && field.row < Number(moveRow) && (field.columnIndex - columnInd) === (Number(moveRow) - field.row))));
+                let startBishopPosition = fourBlocks.filter(field => field.piece === 'B');
+                startBishopPosition[0].piece = '';
+                let newBishopPos = fields.filter(field => field.columnIndex === columnInd && field.row === Number(moveRow));
+                newBishopPos[0].piece = 'B';
+            } else if(piece === 'e'){
+                var startPawnPosition = fields.filter(field => ((field.column === piece) && (field.row === (Number(moveRow) - 1))));
+                startPawnPosition[0].piece = '';
+                var eatPosition = fields.filter(field => field.column === moveColumn && field.row === Number(moveRow));
+                eatPosition[0].piece = 'P';
+            }
+        } else {
+            if(piece === 'q') {              
+                let allQueens = fields.filter(field => field.piece === 'q');
+                allQueens[0].piece = '';
+                let newColumnFields = fields.filter(field => field.column === moveColumn);
+                let newPosition = newColumnFields.filter(field => field.row === Number(moveRow));
+                newPosition[0].piece = 'q';
```

Comment:
This line actually not doing anything in code? If you delete every line like this one it will work still because this variable is not used at all?